srcpkgs/forgejo/files/forgejo/run

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+set -e
+
+export USER=_forgejo
+export HOME=/var/lib/forgejo
+
+export GITEA_WORK_DIR="${HOME}"
```

Comment:
is it really still called GITEA_WORK_DIR?

Reply:
afaik yes, many things were left unchanged to enable easier migration from gitea

Reply:
> is it really still called GITEA_WORK_DIR?

no, it is: `$FORGEJO_WORK_DIR`

- https://forgejo.org/docs/latest/admin/config-cheat-sheet/#default-configuration-non-appini-configuration